modules/gis/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@loaders.gl/gis",
+  "description": "Helpers for GIS category data",
+  "version": "2.1.0-beta.2",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/uber-web/loaders.gl"
+  },
+  "keywords": [
+    "geometry",
+    "GeoJSON"
+  ],
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "esnext": "dist/es6/index.js",
+  "sideEffects": false,
+  "files": [
+    "src",
+    "dist",
+    "README.md"
+  ],
+  "scripts": {
```

Comment:
We can remove the worker scripts for now (at the moment only loaders have workers). This is also why your CI builds are failing.